content/docs/command-reference/remote/add.md

```diff
@@ -104,32 +104,12 @@ By default, DVC authenticates using your AWS CLI
 (if set). This uses the default AWS credentials file. To use a custom
 authentication method, use the parameters described in `dvc remote modify`.
 
-S3 remotes can also be configured entirely via environment variables, e.g.:
+The following API methods are performed by DVC: `list_objects_v2` or
+`list_objects`, `head_object`, `download_file`, `upload_file`, `delete_object`,
+`copy`. So make sure you have the following permissions enabled:
```

Comment:
let's start with "Make sure " (and then mention API calls) - it's more practical to my mind.

Reply:
Done!